src/component/Header/MenuBar/MenuBar.jsx

```diff
@@ -0,0 +1,19 @@
+import { Button, Toolbar } from '@material-ui/core';
+import HomeIcon from '@material-ui/icons/Home';
+import React from 'react';
+import Explore from './Explore';
+import Music from './Music';
+
+const MenuBar = () => (
+  <div>
```

Comment:
any reason you keep wrapping everything around a `<div>` instead of a fragment `<>` ?

Reply:
no reason for using div. i will do empty frangment.